### PR TITLE
Us01 onboarding screen

### PR DESCRIPTION
**Pull request by thanhphamtrung**

### CODE REVIEW

File: lib/src/screens/onboarding_screen.dart [hunk 1]
```diff
@@ -6,12 +6,157 @@ import '../widgets/logo.dart';
...
   @override
   Widget build(BuildContext context) {
-    return Container(
-      
+    double widthsizebutton = 239;
+    double heightsizebutton = 50;
+    double spacingLogoWithAppBar = 80;
+    double spacingTitleWithSubTitle = 8;
+    double spacingSubTitleWithButton = 34;
+    double spacingButton = 20;
```
Comment (thanhphamtrung) [inline lib/src/screens/onboarding_screen.dart:16]: spacing too much

File: pubspec.lock [hunk 2]
```diff
@@ -1,20 +1,69 @@
 # Generated by pub
```
Comment (thanhphamtrung) [inline pubspec.lock:1]: this file is generated automatically so we shouldn't commit it. this time is oke

File: test/screens/onboarding_screen_test.dart [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  // group('tablet onboarding screen', () {
+  //   testWidgets('Should render screen with title', (tester) async {
+  //     tester.binding.window.physicalSizeTestValue = Size(1550, 500);
+  //     addTearDown(tester.binding.window.clearPhysicalSizeTestValue);
+  //     await tester.pumpWidget(widget);
+  //     final titleApp = find.text(
+  //         'Never run out of ideas again. Try new foods,\n ingredients, cooking style, and more');
+  //     expect(titleApp, findsOneWidget);
+  //   });
+  // });
```
Comment (thanhphamtrung) [inline test/screens/onboarding_screen_test.dart:28]: is it ready for review?

Comment (TDKBao) [reply test/screens/onboarding_screen_test.dart:28]: not yet. It's a testing widget, I'm still in progress testing. Just push on branch to prepare to work.

File: pubspec.lock (hunk 2 above)
Comment (TDKBao) [reply pubspec.lock:1]: That was my mistake, I forgot to check commit file, next time I will remove it when commit. Thank you

**Pull request merged**